vendor/modules.txt

```diff
@@ -531,9 +531,8 @@ k8s.io/client-go/util/workqueue
 k8s.io/component-base/metrics
 k8s.io/component-base/version
 # k8s.io/klog v1.0.0
-## explicit
 k8s.io/klog
-# k8s.io/klog/v2 v2.2.0
+# k8s.io/klog/v2 v2.3.0
```

Comment:
k8s.io/klog is still being pulled in by `vendor/github.com/kubernetes-csi/csi-lib-utils/connection/connection.go`.

Jan is working on a new release of csi-lib-utils which uses klog/v2. We need to update to that upcoming release.

Reply:
ok

Reply:
Can we get this PR merged first?  Then a followup PR can be done to update csi-lib-utils?

Reply:
If we merge this PR as-is, we loose log output from csi-lib-utils because it then logs via klog/v1 with default log level.

We really should switch everything at once.

Reply:
I'm ok merging it in phases as long as we don't release without it.

Reply:
I won't cut a release without the csi-lib-utils change.  I see that there are 2 PRs in external-attacher to complete this change as well.